app/mailers/notify_mailer.rb

```diff
@@ -4,6 +4,9 @@ class NotifyMailer < Mail::Notify::Mailer
 
   include Devise::Controllers::UrlHelpers
 
+  def default_url_options
+    Rails.application.config.action_mailer.default_url_options
```

Comment:
This should be equivelant and a bit more concise:
```suggestion
    Rails.configuration.action_mailer.default_url_options
```